backend/src/main/java/com/woowacourse/levellog/team/exception/InterviewCloseException.java

```diff
@@ -0,0 +1,11 @@
+package com.woowacourse.levellog.team.exception;
+
+import com.woowacourse.levellog.common.exception.LevellogException;
+import org.springframework.http.HttpStatus;
+
+public class InterviewCloseException extends LevellogException {
```

Comment:
```suggestion
public class InterviewTimeException extends LevellogException {
```
좀 더 general한 이름이 좋지않을까요?